src/runtime.js

```diff
@@ -22,18 +22,20 @@ socket.events.on('offline', () => {
 module.exports.start = options => {
   options = options || {};
   config.host = options.host || config.host;
-  if (options.uuid && options.secret) {
-    credentials.setDeviceCredentials(options.uuid, options.secret);
+  if ((options.uuid || options.deviceUuid) && options.secret) {
+    credentials.setDeviceCredentials(options.uuid || options.deviceUuid, options.secret);
+  } else if (options.networkUuid && options.apiKey) {
+    credentials.setNetworkCredentials(options.networkUuid, options.apiKey);
```

Comment:
Ya know, I just now noticed that we are calling it a newtork uuid and api key. 
I'd prefer to back up and use networkSecret, or apiSecret. API key is probably going to be something else some day. Your call.